backend/tests/clubs/test_commands.py

```diff
@@ -2,18 +2,71 @@
 import datetime
 import os
 import tempfile
+from unittest import mock
 
+from dateutil.tz import tzutc
 from django.contrib.auth import get_user_model
 from django.core import mail
 from django.core.management import call_command
 from django.core.management.base import CommandError
 from django.test import TestCase
 from django.utils import timezone
+from ics import Calendar
+from ics import Event as Ev
```

Comment:
Ideally make this more descriptive, `CalendarEvent` or `ICSEvent` instead of `Ev`.